### DISCUSSION TITLE
老哥我想问下OCR文本后处理的 单栏-总是换行的具体代码设置是什么

### DISCUSSION THREAD

**Discussion by DawnCx330** [hiroi-sora/Umi-OCR#753] [Q&A]
![Uploading 微信图片_20241226161210.png…]()

**Comment by hiroi-sora** [comment]
https://github.com/hiroi-sora/Umi-OCR/blob/main/UmiOCR-data/py_src/ocr/tbpu/parser_single_line.py

**Reply by H-R-xin** [reply]
well，i think ，the art of genius
